test/react-web/client/graphql/queries-1.test.tsx

```diff
@@ -370,6 +370,44 @@ describe('queries', () => {
     }, 25);
   });
 
+  it('continues to not subscribe to a skipped query when props change', (done) => {
+    const query = gql`query people { allPeople(first: 1) { people { name } } }`;
+    const networkInterface = mockNetworkInterface();
+    const oldQuery = networkInterface.query;
+
+    networkInterface.query = (request) => {
+      console.log('failing')
+      fail(new Error('query ran even though skip present'));
+      return oldQuery(request);
```

Comment:
Lack of preserving `this` caused problems when I copied this into other test. fixed.